src/modules/Common/components/Collections.tsx

```diff
@@ -14,7 +15,8 @@ import { useTranslation } from 'modules/I18n';
 
 type CollectionsProps = {
   ids?: Array<string>;
-  withPlaceholder?: Boolean,
+  withPlaceholder?: Boolean;
+  cardLink?: 'versions' | 'datasets';
```

Comment:
Since it's per collection, there is only one dataset 🙂 
> If you prefer to keep the same plurality, we could maybe use “history” instead of “versions”. Both are fine for me.

```suggestion
  cardLink?: 'versions' | 'dataset';
```